content/blog/2021/machine-learning.md

```diff
@@ -0,0 +1,63 @@
+Title: Machine learning: A major cause of the Pythonic buzz
+Date: 2021-08-24 12:05
+Author: Anuran Roy (@anuran-roy)
```

Comment:
```suggestion
Author: Anuran Roy (@anuran-roy)
```

Reply:
We keep the author fields empty, attribute the content to PyCon India Content Team as whole. So you can remove this.

Reply:
Yeah sure. I'll do the needful.

Reply:
Done.